contribs/gmf/src/directives/partials/gmflayertree.html

```diff
@@ -0,0 +1,19 @@
+<span ng-if="::!layertreeCtrl.isRoot"
+    ng-model="::gmfLayertreeCtrl.getStyleGetterSetter(layertreeCtrl)"
+    ng-model-options="{getterSetter: true}">
+  <img ng-src="{{::layertreeCtrl.node.icon}}"></img>
+  <input type="checkbox"
```

Comment:
look into luxembourg `catalog` how it's done.
https://github.com/Geoportail-Luxembourg/geoportailv3/tree/master/geoportailv3/static/js/catalog